installer/roles/mobile-control-panel-setup/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
```

Comment:
@PhilipGough @maleck13 Thinking about this. Not sure it's the way to go. There are libraries in openshift-ansible that would be good to leverage. In particular these two:

* https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_facts
* https://github.com/openshift/openshift-ansible/tree/master/roles/ansible_service_broker

I think reaching out to them and finding a way to have these accessible through Ansible Galaxy would be a better approach so we can add them to requirements.yml as dependencies.

I'm not sure if it is possible to publish multiple roles in Ansible Galaxy without ending up pulling in every role in this repo currently which is not great.

I think if we need to use these libraries at the moment we can just move the role out to a repo and reference that for the moment.